starknet-rpc-test/src/constants.rs

```diff
@@ -32,4 +32,4 @@ pub const MAX_FEE_OVERRIDE: &str = "0x100000";
 // Need to use `from_mont` because this needs to be a constant function call
 /// ChainId for Starknet Goerli testnet
 pub const SN_GOERLI_CHAIN_ID: FieldElement =
-    FieldElement::from_mont([3753493103916128178, 18446744073709548950, 18446744073709551615, 398700013197595345]);
+    FieldElement::from_mont([3753493103916128178, 18446744073709548950, 18446744073709551615, 398700013197595345]);
```

Comment:
Can't you import it? Rather than defining it again?

Reply:
That was in a previous commit so I think I'll fuck it up if try to recover changes on it

Reply:
Right now it looks like the change is not on your branch. But it should be.

Or am I missing something?